src/api-umbrella/web-app/app/controllers/api/v1/users_controller.rb

```diff
@@ -92,14 +92,12 @@ def update
     end
   end
 
-  def destroy do
+  def destroy
     @api_user = ApiUser.find(params[:id])
+    authorize(@api_user)
     @api_user.enabled = false
-    if(@api_user.save)
-      head :no_content
-    else
-      format.json { render(:json => errors_response(@api_user), :status => :unprocessable_entity) }
-    end
+    @api_user.save
+    respond_with(:api_v1, @api_user, :root => "api_user")
```

Comment:
I intended this to be a 201 no content on success, and the best error on failure would either be a 500 or 422. What do you think?

Reply:
Also, I'm not a rails guy either ;-), I was looking to other parts of the code and they were using this line. Anyway, I've tested it and it is returning a 204 no content response :).